### PR TITLE
AGPL-1.0: Deprecate AGPL-1.0 in favor of -only and -or-later forms

### LINKED ISSUE #542: add GNU licenses with new identifiers (*-only and *-or-later)
* GPL-1.0-only or GPL-1.0-or-later 
* GPL-2.0-only or GPL-2.0-or-later
* GPL-3.0-only or GPL-3.0-or-later

* LGPL-2.0-only or LGPL-2.0-or-later
* LGPL-2.1-only or LGPL-2.1-or-later
* LGPL-3.0-only or LGPL-3.0-or-later

* FDL-1.1-only or FDL-1.1-or-later
* FDL-1.2-only or FDL-1.2-or-later
* FDL-1.3-only or FDL-1.3-or-later

* AGPL-3.0-only or AGPL-3.0-or-later

license text is same for each set
ensure standard header reflect correct wording for each one

### LINKED ISSUE #543: deprecate GNU licenses with current identifiers
GPL-1.0
GPL-2.0
GPL-3.0

LGPL-2.0
LGPL-2.1
LGPL-3.0

FDL-1.1
FDL-1.2
FDL-1.3

AGPL-3.0

### PR DESCRIPTION
**Pull request by wking**
[As][1] [requested][2] by @jlovejoy.

I've also stripped trailing whitespace from the `AGPL-1.0` to avoid introducing trailing whitespace when I copied it into the new files before updating them.  This keeps the diffs minimal for easy review:

```console
$ diff -u1 AGPL-1.0.xml AGPL-1.0-or-later.xml
--- AGPL-1.0.xml        2018-01-11 16:46:38.922506692 -0800
+++ AGPL-1.0-or-later.xml       2018-01-11 16:46:38.920506692 -0800
@@ -3,4 +3,3 @@
    <license isOsiApproved="false" licenseId="AGPL-1.0"
-            name="Affero General Public License v1.0"
-            isDeprecated="true" deprecatedVersion="3.1">
+            name="Affero General Public License v1.0 or later">
       <crossRefs>
@@ -8,3 +7,5 @@
       </crossRefs>
-      <notes><p>DEPRECATED: Use the license identifier AGPL-1.0-only instead of APL-1.0, and use AGPL-1.0-or-later instead of AGPL-1.0.</p></notes>
+      <notes>
+        <p>Section 9 of this license allows content under this "any later version" grant to be redistributed under the GPL-3.0-or-later.  Affero Inc. also released an AGPL-2.0 (http://www.affero.org/agpl2.html) to allow AGPL-1.0-or-later work to be distributed under the AGPL-3.0-or-later.</p>
+      </notes>
       <titleText>
$ diff -u1 AGPL-1.0.xml AGPL-1.0-only.xml
--- AGPL-1.0.xml        2018-01-11 16:46:38.922506692 -0800
+++ AGPL-1.0-only.xml   2018-01-11 16:46:38.916506693 -0800
@@ -3,4 +3,3 @@
    <license isOsiApproved="false" licenseId="AGPL-1.0"
-            name="Affero General Public License v1.0"
-            isDeprecated="true" deprecatedVersion="3.1">
+            name="Affero General Public License v1.0 only">
       <crossRefs>
@@ -8,3 +7,2 @@
       </crossRefs>
-      <notes><p>DEPRECATED: Use the license identifier AGPL-1.0-only instead of APL-1.0, and use AGPL-1.0-or-later instead of AGPL-1.0.</p></notes>
       <titleText>

```

I've also added a note pointing out that both `AGPL-3.0-or-later` and `GPL-3.0-or-later` are available to folks redistributing `AGPL-1.0-or-later` content (although I think we'll also want the spec update for `+` that I've floated [here][3]).

Fixes #542.
Fixes #543.

[1]: https://github.com/spdx/license-list-XML/issues/542#issuecomment-357074043
[2]: https://github.com/spdx/license-list-XML/issues/543#issuecomment-357074182
[3]: https://github.com/spdx/spdx-spec/pull/37#issuecomment-354325141

### CODE REVIEW

**Comment by wking** [comment]
Rebased around #452 with 238513c → 4aa72b8.

**Comment by wking** [comment]
@goneall, do you understand what's happening here?  spdx-tools is [complaining about][1]:

    Duplicates licenses: AGPL-1.0, AGPL-1.0

but I'm trying to follow the pattern used by the other GPLs in [my `expected-warnings` change][2].

[1]: https://travis-ci.org/spdx/license-list-XML/jobs/332081076#L481
[2]: https://github.com/spdx/license-list-XML/pull/599/files#diff-d00b85a566a4344b53d0a41c7ef266cfL1

**Comment by wking** [comment]
Dp you understand why the tools have the same ID twice?  The other dups have different IDs.

File: src/AGPL-1.0-only.xml [hunk 1]
```diff
@@ -0,0 +1,276 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+   <license isOsiApproved="false" licenseId="AGPL-1.0"
```
Comment (wking) [inline src/AGPL-1.0-only.xml]: Ah, need to update `licenseId`.

Comment (wking) [reply src/AGPL-1.0-only.xml]: > Ah, need to update `licenseId`.

Done with 373bf50 → 96f8f64

**Comment by goneall** [comment]
> Do you understand why the tools have the same ID twice? The other dups have different IDs.

I didn't even notice the duplicate ID's - just copy/pasted.  It should have different ID's - one for the deprecated license ID and one for the new license ID.  I'll see if I can figure it out.

**Pull request merged**